dog-blog-page3.html

```diff
@@ -0,0 +1,61 @@
+    <title>Adopt-a-Dog</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <!--this is for the future style sheet-->
+    <link rel="stylesheet" type="text/css" href="styles.css">
+  </head>
+  <body>
+    <header>
+      <h1>ADOPTADOG</h1>
+        <img src="adoptadog-logo.svg" width="500" alt="white dog silhoutte">
+        <h2>Seattle's Premier Dog Adoption Agency</h2>
+        <ing src="cart-icon.svg" width="200" alt="cart icon">
+          <a href="dog-checkout-page4.html">Checkout</a>
+      </header>
+      <nav>
+        <ul>
+          <li>Home</li>
+          <li>Dogs</li>
+          <li>Adopt</li>
+        </ul>
+      </nav>
+      <div>
+        <h2>Adoptadog DogBlog</h2>
+      </div>
+        <div>
+          <img src="blog-1.jpg" width="500" alt="a pet owner hugs his dog while both look at a grand desert vista">
+            <h3>The Best Ways to Keep Your Travel-happy</h3>
+               <p>Pet Food pet supplies gimme five puppy cage food feathers food heel feathers running pet gate walk lazy dog Spike. Good Boy park lazy dog walk kibble Scooby snacks licks canary. Roll Over kitty barky critters litter stick window litter box wagging field toy. Whiskers harness biscuit food lick small animals throw meow house train. Bedding field hamster small animals carrier polydactyl groom vaccine. Commands running gimme five groom slobber run fast head ball litter box biscuit catch run fast roll over. Roll Over litter box tabby pet slobbery play dead kitty roll over small animals barky good boy string kitty fish licks teeth chew drool. ID Tag barky lick parakeet wet nose ball walk tabby wag tail chirp nest. Spike bird seed water teeth leash ball. Crate pet food stripes carrier drool slobbery tabby dog puppy birds walk roll over bird food Mittens pet supplies.
+               </p>
+         </div>
+         <div>
+           <img src="blog-2.jpg" width="500" alt="">
+             <h3>How to Walk Mulitple Dogs</h3>
+                <p>Pet Food pet supplies gimme five puppy cage food feathers food heel feathers running pet gate walk lazy dog Spike. Good Boy park lazy dog walk kibble Scooby snacks licks canary. Roll Over kitty barky critters litter stick window litter box wagging field toy. Whiskers harness biscuit food lick small animals throw meow house train. Bedding field hamster small animals carrier polydactyl groom vaccine. Commands running gimme five groom slobber run fast head ball litter box biscuit catch run fast roll over. Roll Over litter box tabby pet slobbery play dead kitty roll over small animals barky good boy string kitty fish licks teeth chew drool. ID Tag barky lick parakeet wet nose ball walk tabby wag tail chirp nest. Spike bird seed water teeth leash ball. Crate pet food stripes carrier drool slobbery tabby dog puppy birds walk roll over bird food Mittens pet supplies.
+                </p>
+          </div>
+          <div>
```

Comment:
these blog post div's might be best served by an HTML5 element. Maybe something you'd read....